src/convertToObject.js

```diff
@@ -5,8 +5,48 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = function (sourceString) {
+  const styleObject = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `styleObject`. Use more descriptive names to increase the readability of your code.